core/metrics/exporter.go

```diff
@@ -0,0 +1,222 @@
+	prometheus.MustRegister(metrics.storesGauge)
+	prometheus.MustRegister(metrics.jobsGauge)
+	prometheus.MustRegister(metrics.tasksGauge)
+	prometheus.MustRegister(metrics.archivesGauge)
+	prometheus.MustRegister(metrics.storageUsedBytesGauge)
+
+	metrics.tenantsGauge.Set(float64(config.TenantCount))
+	metrics.agentsGauge.Set(float64(config.AgentCount))
+	metrics.targetsGauge.Set(float64(config.TargetCount))
+	metrics.storesGauge.Set(float64(config.StoreCount))
+	metrics.jobsGauge.Set(float64(config.JobCount))
+	metrics.tasksGauge.Set(float64(config.TaskCount))
+	metrics.archivesGauge.Set(float64(config.ArchiveCount))
+	metrics.storageUsedBytesGauge.Set(float64(config.StorageUsedCount))
+
+	return metrics
+}
+
+func (m *Metrics) Inform(mbus *bus.Bus) {
+	m.bus = mbus
+}
+
+func (m *Metrics) ServeExporter() http.Handler {
+	return promhttp.Handler()
+}
+
+func (m *Metrics) CreateObjectCount(typeThing string, data interface{}) {
+	interfaceData := data.(map[string]interface{})
+	switch typeThing {
+	case "tenant":
+		m.tenantsGauge.Inc()
+	case "agent":
+		m.agentsGauge.Inc()
+	case "target":
+		m.targetsGauge.Inc()
+	case "store":
+		m.storesGauge.Inc()
+	case "job":
+		m.jobsGauge.Inc()
+	case "task":
+		m.tasksGauge.Inc()
+	case "archive":
+		m.archivesGauge.Inc()
+		m.storageUsedBytesGauge.Add(float64(interfaceData["size"].(int64)))
+	default:
+		log.Infof("Event type not recognized or not implemented yet.")
+	}
+}
+
+func (m *Metrics) UpdateObjectCount(typeThing string, data interface{}) {
+	interfaceData := data.(map[string]interface{})
+	switch typeThing {
+	case "archive":
+		if interfaceData["status"] == "manually purged" {
+			m.archivesGauge.Dec()
+			m.storageUsedBytesGauge.Sub(float64(interfaceData["size"].(int64)))
+		}
+	}
+}
+
+func (m *Metrics) DeleteObjectCount(typeThing string) {
+	switch typeThing {
+	case "tenant":
+		m.tenantsGauge.Dec()
+	case "agent":
+		m.agentsGauge.Dec()
+	case "target":
+		m.targetsGauge.Dec()
+	case "store":
+		m.storesGauge.Dec()
+	case "job":
+		m.jobsGauge.Dec()
+	default:
+		log.Infof("Event type not recognized or not implemented yet.")
+	}
+}
+
+/* TODO
+func (m *Metrics) UpdateCoreStatus(value float64) {
+	coreStatusGauge.Set(value)
+} */
+
+func (m *Metrics) RegisterBusEvents(queues ...string) {
+	ch, _, err := m.bus.Register(queues)
+	if err != nil {
+		log.Infof("bus didn't register for mbus")
+		return
+	}
+
+	for eventObject := range ch {
+		event := eventObject.Event
+		typeThing := eventObject.Type
+		var data interface{} = eventObject.Data
+		data = eventObject.Data
+		if event == "lock-core" {
+			log.Infof("TODO")
+		} else if event == "unlock-core" {
+			log.Infof("TODO")
+		} else if event == "create-object" {
+			m.CreateObjectCount(typeThing, data)
+		} else if event == "update-object" {
+			m.UpdateObjectCount(typeThing, data)
+		} else if event == "delete-object" {
+			m.DeleteObjectCount(typeThing)
+		} else {
+			log.Infof("Event not recognized yet")
```

Comment:
``Metrics ignoring event of type `%s'``, where `%s` is the `event` variable.
```suggestion
			log.Debugf("Metrics ignoring event of type `%s'", event)
```

Reply:
Change to a debug level log.